packages/flutter/lib/src/material/bottom_app_bar.dart

```diff
@@ -135,6 +137,12 @@ class BottomAppBar extends StatefulWidget {
   /// See [Material.surfaceTintColor] for more details on how this overlay is applied.
   final Color? surfaceTintColor;
 
+  /// The color used as a drop-shadow, if the app bar is elevated.
+  ///
+  /// If this is null, the value will depend whether Material 3 is used - see [Material.shadowColor].
+  /// In Material 3, this value defaults to [Colors.transparent].
+  final Color? shadowColor;
+
```

Comment:
for consistency, please use the documentation for existing `shadowColor` properties as inspo

Reply:
Sure - took some artistic license from `app_bar.dart`. Changed `bottom_sheet.dart` too.